pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -1,45 +1,56 @@
-#' Read input folder of 10x data
+#' Read user input files
+#'
+#' Checks technology used and dispatches call to correct reading function.
+#' In case of 10x, `annot` contains a data.frame from reading the features.tsv.gz file,
+#' while in rhapsody it contains the gene symbols as read from the count matrix.
 #'
 #' @inheritParams download_user_files
 #' @param prev_out list with experiment configuration settings
 #'
 #' @return list with 'output' slot containing \itemize{
 #'   \item{"counts_list"}{named list of dgCMatrix per sample}
-#'   \item{"annot"}{data.frame from reading features.tsv.gz for each sample}
-#' }
+#'   \item{"annot"}{data.frame with gene ids and/or symbols}
 #' @export
-#'
-load_cellranger_files <- function(input, pipeline_config, prev_out, input_dir = "/input") {
-  message("Loading cellranger output ...")
+load_user_files <- function(input, pipeline_config, prev_out, input_dir = "/input") {
+  message("Loading user files...")
   check_prev_out(prev_out, "config")
 
   # destructure previous output
   config <- prev_out$config
 
-  output <- c(prev_out, call_read10x(config, input_dir))
+  technology <- config$input$type
```

Comment:
Are we sure type will be set in all cases? We haven't been using this so far and I fear it might be null. Maybe solve this with if tech == rhapsody do rhapsody, else do 10x.